packages/trader/src/AppV2/Components/ActiveSymbolsList/__tests__/active-symbols-list.spec.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { screen, render } from '@testing-library/react';
+import ActiveSymbolsList from '../active-symbols-list';
+import TraderProviders from '../../../../trader-providers';
+import { TCoreStores } from '@deriv/stores/types';
+import { mockStore } from '@deriv/stores';
+
+jest.mock('AppV2/Components/SymbolSearchResults', () => jest.fn(() => <div>MockedSymbolSearchResults</div>));
+jest.mock('AppV2/Components/SymbolsSearchField', () => jest.fn(() => <div>MockedSymbolsSearchField</div>));
+jest.mock('AppV2/Components/MarketCategories', () => jest.fn(() => <div>MockedMarketCategories</div>));
+
+jest.mock('AppV2/Hooks/useActiveSymbols', () => ({
+    ...jest.requireActual('AppV2/Hooks/useActiveSymbols'),
+    __esModule: true,
+    default: jest.fn(() => ({
+        default_symbol: 'CADAUD',
+        activeSymbols: [],
+        fetchActiveSymbols: jest.fn(),
+    })),
+}));
+
+describe('<ActiveSymbolsList />', () => {
+    const mocked_props = {
+        isOpen: true,
+        setIsOpen: jest.fn(),
+    };
+    const mocked_store = {
+        modules: {
+            trade: {
+                symbol: '',
+            },
+        },
+    };
+    const MockActiveSymbolsList = (
+        mocked_store: TCoreStores,
+        mocked_props: Parameters<typeof ActiveSymbolsList>[0]
+    ) => {
+        return (
+            <TraderProviders store={mocked_store}>
+                <ActiveSymbolsList {...mocked_props} />
+            </TraderProviders>
+        );
+    };
+    it('renders SymbolsSearchField', () => {
+        render(MockActiveSymbolsList(mockStore(mocked_store), mocked_props));
```

Comment:
Absolutely optional suggestion: you can omit passing `mockStore(mocked_store)` and `mocked_props` as you are not changing them